src/main/java/com/redhat/rhjmc/containerjfr/MainModule.java

```diff
@@ -89,6 +89,17 @@ public static Gson provideGson() {
     @Provides
     @Named(RECORDINGS_PATH)
     static Path provideSavedRecordingsPath() {
-        return Paths.get("/", "flightrecordings");
+        String ARCHIVE_PATH = null;
+        final Logger logger = provideLogger();
```

Comment:
You should be able to have Dagger inject a Logger instance into this method as a parameter, rather than having to grab the logger by manually calling this method like this. Just add a Logger to the method's argument list.

Reply:
You can see examples of this in other DI modules around the codebase, for example the `NetworkModule`.

Reply:
Ok, fixed!